_posts/research/2020-04-22-cooperative_multirobot_systems.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Autonomous Coordination of Cooperative Multi-Robot Systems
+description: Multi-Agent Reinforcement Learning for Robot Control
+author: Joewie J. Koh and Guohui Ding
+permalink: research/cooperative_multirobot_systems.html
+image:
+    feature: research/multirobot/cooperative_manipulation.png
+    size: 75%
+excerpt_separator: <!-- More -->
+---
+
+Heterogeneous teams of cooperative multi-robot systems have the potential to propel robotics at large toward a safer, more accessible, and lower cost future.
+Through distribution of roles among the various agents, a multi-robot system can be imbued with capabilities that are more than the sum of its parts by coordinating multiple agents to cooperatively accomplish common goals; using search and rescue as an example, cooperation between unmanned aerial and ground vehicles could increase the odds of success with larger areas covered, better mobility.
+However, autonomous coordination of robots cannot rely on predefined rules or simple control schemes.
```

Comment:
This is a negative sentence, switch it to positive. Predefined rules or simple control schemes is not how MRS are controlled and if you say so you basically neglect the research from thousands people.
Tbh I think the previous paragraph you had was a better tribute to prior work and was working better.